src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -380,6 +395,12 @@ export const handleCountFormSubmissions: RequestHandler<
     })
 }
 
+// Handler for GET /admin/forms/:formId/submissions/count
+export const handleCountFormSubmissions = [
```

Comment:
nit: instead of casting, can declare the type explicitly

```ts
export const handleCountFormSubmissions: RequestHandler[] = [
  ...
]
```

Reply:
good point, will do so

Reply:
as per our discussion on slack, because `RequestHandler` requires a `ParamsDictionary` as its first type, this becomes alot harder to do without casting because ParamsDictionary isn't generic. will just cast :/